src/main/kotlin/org/rust/lang/core/resolve/RustResolveEngine.kt

```diff
@@ -280,8 +281,11 @@ private class Resolver {
 
         override var matched: RustNamedElement? = null
 
-        override fun visitModItem(o: RustModItem) {
-            seek(o.itemList)
+        override fun visitModItem(mod: RustModItem) {
+            seek(mod.itemList)
+            if (matched == null) {
+                seekInjectedItems(mod)
```

Comment:
Besides `extern crate std` in the root module, `use std::prelude::*` is injected in *every* module. I haven't handled this yet because I want to deal with usual star imports first.